pkg/codec/opus/mixer.go

```diff
@@ -0,0 +1,44 @@
+package opus
+
+import (
+	"errors"
+
+	"github.com/pion/mediadevices/pkg/wave"
+)
+
+// ChannelMixer mixes audio into specifix channels.
+type ChannelMixer interface {
+	Mix(dst wave.Audio, src wave.Audio) error
+}
+
+// MonoMixer mixes channels into monaural audio.
+type MonoMixer struct {
+}
+
+func (m *MonoMixer) Mix(dst wave.Audio, src wave.Audio) error {
+	if dst.ChunkInfo().Len != src.ChunkInfo().Len {
+		return errors.New("buffer size mismatch")
+	}
+	dstSetter, ok := dst.(interface {
+		Set(i, ch int, s wave.Sample)
+	})
```

Comment:
I think we should define the interface similarly to how `image/draw` package does it. So, maybe something like:

```go
type Audio interface {
    wave.Audio
    Set(i, ch int, s wave.Sample)
}
```

Reply:
Hmm, I don't have enough vocabulary for naming the package.
My current candidates are `wave/sound` (might be too general) and `wave/dsp` (sounds too high layer).

Reply:
Hmm, how about if we put it in `wave` still, but the interface is `EdittableAudio`?

Reply:
Yea, it sounds better.